tasks/terraform-cli/src/tests/features/workspace/terraform-workspace-new.feature

```diff
@@ -28,3 +28,12 @@ Feature: terraform workspace new
     When the terraform cli task is run
     Then the terraform cli task executed command "terraform workspace new bar"
     And the terraform cli task fails with message "\u001b[31mWorkspace \"bar\" already exists\u001b[0m\u001b[0m\n"
+
+    Scenario: new workspace already exists but it's allowed

```

Comment:
Could you also add a sample pipeline to the test pipelines? You could probably update the [`switch_workspaces.yaml`](https://github.com/ForesightMiningSoftwareCorporation/azure-pipelines-tasks-terraform/blob/main/pipelines/test/switch_workspaces.yml) test to run the `workspace new` command twice. The second instance could have this option configured to ignore. In that case we should not see a failure.